apps/web/src/modules/transaction/pages/casting/components/ConfirmModalBody/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const MODAL_OPTION = {
+  accept: {
+    title: 'ทำเครื่องหมายว่าเสร็จสิ้นแล้ว',
+    finalState: 'จัดการเสร็จสิ้น',
+  },
+  reject: {
+    title: 'ปฏิเสธคำขอธุรกรรม',
+    finalState: 'ถูกปฏิเสธ',
+  },
+} as const
```

Comment:
why `as const` ?

Reply:
At first, I try to make the menubar type-strict which I have given up now 😂